dbms/src/Compression/CompressionCodecZSTD.cpp

```diff
@@ -63,7 +63,7 @@ CompressionCodecZSTD::CompressionCodecZSTD(int level_)
 
 void registerCodecZSTD(CompressionCodecFactory & factory)
 {
-    UInt8 method_code = static_cast<char>(CompressionMethodByte::ZSTD);
+    UInt8 method_code = UInt8(CompressionMethodByte::ZSTD);
```

Comment:
C-style cast doesn't look good

Reply:
This is not a C-style cast. It is named "constructor cast".